lib/Cake/Core/App.php

```diff
@@ -889,6 +889,31 @@ protected static function _packageFormat() {
 	}
 
 /**
+ * Increases the PHP "memory_limit" ini setting by the specified amount
+ * in megabytes  
+ *  
+ * @param string $additionalMb Number in megabytes
+ * @return void
+ */
+	public static function increasePhpMemoryLimit($additionalMb) {
```

Comment:
There's also `K` (kilobytes), and all other units (including none), are treated as bytes. Also the units are case-insensitive, ie you may receive `1m`. See http://php.net/manual/en/function.ini-get.php#refsect1-function.ini-get-examples for an example of how to convert the returned value.